### DISCUSSION TITLE
Support for Local Intros Plugin

### DISCUSSION THREAD

**Discussion by brkr1** [DonutWare/Fladder#143] [Ideas]
### Summary

I would like to request the integration of the [Jellyfin Intros Plugin](https://github.com/dkanada/jellyfin-plugin-intros) into Fladder. 

The Jellyfin Intros Plugin is designed to play custom intro videos before a movie or TV episode starts, similar to how some streaming platforms like Netflix provide a brief introductory clip. 

Integrating this plugin would enhance the user experience by allowing users to see custom intros, such as studio logos or personalized messages, before the main content. 

### Basic Example

When a user selects a movie, the Jellyfin Intros Plugin can automatically play a short, custom video intro (e.g., a studio logo, a personal greeting) before starting the movie.

If the user is watching a TV show, an intro could play before each episode, setting the stage for the viewing experience.

This feature could be customizable, allowing users to upload their own intro files or select from a pre-existing library, giving them control over what plays before content starts.

### Drawbacks

Depending on implementation, integrating the plugin might require additional server resources to manage the playback of intros, especially if they're high-quality videos.

Some users might find the intros repetitive or unnecessary, so it would be crucial to include options to skip or disable them.

If not implemented properly, this feature could impact playback performance, causing delays in starting the main content.

It might introduce a dependency on the Jellyfin Intros Plugin, requiring it to be maintained and updated to ensure compatibility with future versions of the project.

**Comment by coder747-8i** [comment]
That is one of the only issues with this app

**Comment by PartyDonut** [comment]
The intros plugin is archived for quite some time already I might add. 
I'm not sure supporting a plugin that seems to have no more backing is something we should focus on.

Is this already/or going to be replaced with something else?
